### DISCUSSION TITLE
Is it possible to make punctuation only delay dialogue when followed by a space?

### DISCUSSION THREAD

**Discussion by eisegesis-623** [nathanhoad/godot_dialogue_manager#401] [Q&A]
The automatic feature to delay dialogue on certain punctuation marks is absolutely brilliant. That said, there are a few situations where it breaks down a bit.
Ellipsis create a massive pause in dialogue.
`Well... that's a bit awkward.`
For example, in the following line...
`(Overworld descriptions can use parentheses to show that it's not dialogue.)`
...there's a pause between the "." and the ")" at the end.

I'm pretty sure that simply ensuring a punctuation mark is followed by a space would solve this. Is this, or some equivalent solution, currently possible?

**Comment by nathanhoad** [comment]
I've just added the option to specify which characters that, when they follow an auto pause character, will cancel it out. While you're waiting for that to be released you can [make this change yourself](https://github.com/nathanhoad/godot_dialogue_manager/commit/157fc57c62b8613da4e3d3bfc5060119815087ea).

**Reply by eisegesis-623** [reply]
Wow, that was fast! Thanks so much!
